SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005956	Stamina consumption cycle duration of companion is increased by 10 seconds per attribute level.
+SKILL_20151224_005957	If companion HP is less than 50%, activate [응급치료] to recover HP for 2 seconds. The duration of the recovery is increased by 1 second per attribute level. [응급치료] can't be used for another 10 minutes after activation
+SKILL_20151224_005958	Bash: Specialty
+SKILL_20151224_005959	Increases the damage dealt on an enemy with [Bash] by 35 per attribute level.
+SKILL_20151224_005960	Increases the maximum HP of your character by 5% per attribute level while [Swash Buckling] is active.
+SKILL_20151224_005961	Increases your physical defense by 10% per attribute level when [Guard] is active.
+SKILL_20151224_005962	Increases the character's evasion by 6% per attribute level while [Guardian] is active.
+SKILL_20151224_005963	Critical hit attack is increased depending on the physical attack when [Aggressor] is active. Additional value is increased by 5% per attribute level.
+SKILL_20151224_005964	Shield Shoving: Enhance
+SKILL_20151224_005965	Increases the damage dealt on an enemy with [Shield Shoving] by 1% per attribute level.
+SKILL_20151224_005966	Shield Bash: Enhance
+SKILL_20151224_005967	Increases the damage dealt on an enemy with [Shield Bash] by 1% per attribute level.
+SKILL_20151224_005968	Slithering: Additional Damage
+SKILL_20151224_005969	The enemies hit by [Slithering] is vulnerable to [Strike] for 10 seconds and increase the additional damage by 20% per attribute level.
+SKILL_20151224_005970	Targe Smash: Remove Knockback
+SKILL_20151224_005971	[Targe Smash] no longer knockback enemies.
+SKILL_20151224_005972	Increases your block penetration by 10% per attribute level when equipped with a [Two-handed Spear].
+SKILL_20151224_005973	Pistol Shot: Enhance
+SKILL_20151224_005974	Increases the damage dealt on an enemy with [Pistol Shot] by 1% per attribute level.
+SKILL_20151224_005975	Decreases damage taken to x2 from x3 when [Double Pay Earn] is active.
+SKILL_20151224_005976	Zonhau: Enhance
+SKILL_20151224_005977	Increases the damage dealt on an enemy with [Zonhau] by 1% per attribute level.
+SKILL_20151224_005978	Redel: Enhance
+SKILL_20151224_005979	Increases the damage dealt on an enemy with [Redel] by 1% per attribute level.
+SKILL_20151224_005980	Zucken: Enhance
+SKILL_20151224_005981	Increases the damage dealt on an enemy with [Zucken] by 1% per attribute level.
+SKILL_20151224_005982	Cyclone: Resist
+SKILL_20151224_005983	Increase the chance to block harmful effect by 15% per attribute level during [Cyclone].
+SKILL_20151224_005984	STA consumption cycle duration is increased by 0.05 seconds per attribute level when [Bunshin no Jutsu] is active.
```

Comment:
Increases the STA cost cycle duration by 0.05% seconds per attribute level while [Bunshin no Jutsu] is active.